frontend/containers/open-call-page/header/component.tsx

```diff
@@ -0,0 +1,190 @@
+import { OpenCallStatus } from 'enums';
+
+import OpenCallChart from '../chart';
+
+import { OpenCallHeaderProps } from '.';
+
+export const OpenCallHeader: FC<OpenCallHeaderProps> = ({
+  openCall,
+  instrumentTypes,
+  // handleFavorite,
+  // handleApply,
+}) => {
+  const intl = useIntl();
+  const { locale } = useRouter();
+  const {
+    name,
+    instrument_types,
+    maximum_funding_per_project,
+    picture,
+    language,
+    created_at,
+    closing_at,
+    status,
+  } = openCall;
+  const coverImage = picture?.medium || '/images/avatar.svg';
+  const originalLanguage = language || 'es';
+  const totalProjects = 0;
+  const isFavorite = false;
+
+  const openCallRange = useMemo(() => {
+    const openDate = dayjs(created_at);
+    const closingDate = dayjs(closing_at);
+    const deadline = closingDate.format('DD MMM YYYY');
+    // total duration of the open call in days
+    const duration = closingDate.diff(openDate, 'day');
+    // remaining days of the open call. Difference between today and the open call ending date in days
+    const remaining = closingDate.diff(dayjs(), 'day');
+    // consumed days of the open call
+    const consumed = duration - remaining;
+    return { consumed, remaining, deadline };
+  }, [closing_at, created_at]);
+
+  return (
+    <LayoutContainer className="-mt-10 md:mt-0 lg:-mt-16">
+      <Breadcrumbs
+        className="px-4 sm:px-6 lg:px-8"
+        substitutions={{
+          id: { name },
+        }}
+      />
+      <div className="mt-4">
+        <div
+          className="flex items-end mx-4 bg-center bg-cover lg:mx-0 rounded-2xl bg-radial-green-dark bg-green-dark min-h-[250px] lg:min-h-[372px]"
+          style={{
+            ...(coverImage && { backgroundImage: `url(${coverImage})` }),
+          }}
+        >
+          <LayoutContainer>
+            <div className="mb-8 text-center lg:w-1/2 lg:text-left">
+              <h1 className="font-serif text-3xl text-white">{name}</h1>
+            </div>
+          </LayoutContainer>
+        </div>
+        <LayoutContainer className="flex flex-col justify-between mt-8 lg:flex-row">
+          <div className="w-full lg:w-6/12">
+            {originalLanguage && (
+              <span className="block mb-4 text-sm text-gray-400">
+                <FormattedMessage
+                  defaultMessage="Note: The content of this page was originally written in <span>{language}</span>."
+                  id="zXxFL9"
+                  values={{
+                    language: translatedLanguageNameForLocale(intl, originalLanguage),
+                    span: (chunks: string) => <span className="underline">{chunks}</span>,
+                  }}
+                />
+              </span>
+            )}
+          </div>
+          <div className="flex flex-col justify-start lg:mr-4 p-6 bg-white drop-shadow-xl lg:mb-[-70%] h-full lg:translate-y-[-70%] lg:max-w-1/3 rounded-2xl mt-8 lg:mt-0">
```

Comment:
The ring should be displayed on the right hand side, not the left (screenshot from Firefox):
![Progress ring is displayed on the left hand side of the card](https://user-images.githubusercontent.com/6073968/185356777-6a9e6fee-8237-4cc3-87f2-04889cd6bad3.png)